bears/js/JSComplexityBear.py

```diff
@@ -34,7 +34,10 @@ def process_output(self, output, filename, file,
         :param cc_threshold: Threshold value for cyclomatic complexity
         """
         message = '{} has a cyclomatic complexity of {}.'
-        if output:
+        if not output:  # backwards compatible no results
+            return
+
+        if True:  # no else
```

Comment:
`# pragma: no branch`?

Reply:
Could be used, but we'll also get rid of that soon.  This is a more specific rule.